governance/default.go

```diff
@@ -570,14 +570,14 @@ func (g *Governance) initializeCache() error {
 	for _, v := range indices {
 		if num, data, err := g.ReadGovernance(v); err == nil {
 			g.itemCache.Add(getGovernanceCacheKey(num), data)
-			g.actualGovernanceBlock = num
+			atomic.StoreUint64(&g.actualGovernanceBlock, num)
```

Comment:
Is it okay to use `atomic.StoreUint64` and `atomic.LoadUint64`, instead of using `atomic.Value` for `actualGovernanceBlock`?

Reply:
@ehnuje I think the name made you confused. actualGovernanceBlock is the number  of the block and it has uint64 data type.